test/versions.test.js

```diff
@@ -53,4 +53,18 @@ describe('versions', () => {
 
     expect(versionInfo).toEqual(undefined)
   })
+
+  it('extracts a non-semvar version when patchVersions is false', () => {
```

Comment:
```suggestion
  it('extracts a non-semver version when patchVersions is false', () => {
```

Just a minor spelling mistake :)

Reply:
Nicely spotted, I'll fix that!